package.json

```diff
@@ -6,60 +6,54 @@
   "license": "ISC",
   "repository": "https://github.com/probot/webhooks.git",
   "scripts": {
-    "start": "concurrently \"npm run start-dev\" \"npm run watch-ui\"",
-    "start-dev": "nodemon --ignore src/ --ignore dist/ --ignore logs.json ./node_modules/probot/bin/probot-run ./index.js",
-    "start-prod": "probot run ./index.js",
+    "start": "node index.js",
+    "start-dev": "concurrently \"nodemon --ignore src/ ./index.js\" \"webpack -u\"",
```

Comment:
`webpack -w`? I get webpack's usage info printed when I run this.

Reply:
People make typos sometimes. This is one of those times 😂